File: javamodule/src/main/java/com/example/javaModule/SendContactFormUseCase.java
package com.example.javaModule;


import android.os.AsyncTask;

import com.example.domain.domain.models.ContactUsForm;

import java.util.Properties;

import javax.mail.Message;
import javax.mail.MessagingException;
import javax.mail.PasswordAuthentication;
import javax.mail.Session;
import javax.mail.Transport;
import javax.mail.internet.InternetAddress;
import javax.mail.internet.MimeMessage;

public class SendContactFormUseCase {
    private final String sender = "radionovtymur2018@gmail.com";
    private final String senderPassword = "Atlanta2018";

    public void execute(ContactUsForm form) {

        Properties properties = new Properties();
        properties.put("mail.smtp.auth", "true");
        properties.put("mail.smtp.starttls.enable", "true");
        properties.put("mail.smtp.host", "smtp.gmail.com");
        properties.put("mail.smtp.port", "587");
        Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
            @Override
            protected PasswordAuthentication getPasswordAuthentication() {
                return new PasswordAuthentication(sender, senderPassword);
            }
        });
        try {
            Message message = new MimeMessage(session);
            message.setFrom(new InternetAddress(sender));
            String receiver = "radionovtymur@gmail.com";
            message.setRecipients(Message.RecipientType.TO, InternetAddress.parse(receiver));
            message.setSubject("This message is from Diving Travel App");
            message.setText(form.toString());
            Transport.send(message);
        } catch (MessagingException e) {
            e.printStackTrace();
        }
    }
}

File: domain/src/main/java/com/example/domain/domain/useCases/SendContactFormUseCase.kt
package com.example.domain.domain.useCases

import com.example.domain.domain.models.ContactUsForm
import java.util.*
import javax.mail.*
import javax.mail.internet.InternetAddress
import javax.mail.internet.MimeMessage


class SendContactFormUseCase {
    private val sender = "senderEmail@gmail.com"
    private val senderPassword = "senderPassword"
    fun execute (form: ContactUsForm) {
        val properties = Properties()
        properties["mail.smtp.auth"] = "true"
        properties["mail.smtp.starttls.enable"] = "true"
        properties["mail.smtp.host"] = "smtp.gmail.com"
        properties["mail.smtp.port"] = "587"
        Session.getInstance(properties)
        val session: Session = Session.getInstance(properties, object : Authenticator() {
            override fun getPasswordAuthentication(): PasswordAuthentication {
                return PasswordAuthentication(sender, senderPassword)
            }
        })
        try {
            val message: Message = MimeMessage(session)
            message.setFrom(InternetAddress(sender))
            val receiver = "recieverEmail@gmail.com"
            message.setRecipients(Message.RecipientType.TO, InternetAddress.parse(receiver))
            message.subject = "This message is from Diving Travel App"
            message.setText(form.toString())
            Transport.send(message)
        } catch (e: MessagingException) {
            e.printStackTrace()
        }
    }
}